spec/requests/api/suppliers_controller_locations_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::SuppliersController do
+  let(:access_token) { 'spoofed-token' }
+  let(:headers) { { 'CONTENT_TYPE': content_type, 'Authorization': "Bearer #{access_token}" } }
+  let(:content_type) { ApiController::CONTENT_TYPE }
```

Comment:
You can leave out `CONTENT_TYPE` in the headers and the spec will still work - we have coverage for that elsewhere (it's optional) so is just adding noise here. Alternatively we have a shared context you could include that sets all of this up:

```ruby
    include_context 'with supplier with spoofed access token'
```

Reply:
👍 I didn't know that.
Removed!